RIP-3 Nft Bank/RIP-3 Nft Bank.md

```diff
@@ -0,0 +1,81 @@
+---
+rip: 3
+title: Nft Bank
+author:  @butterfiolee
+discussions-to: <URL>
+status: Draft
+created: 2022-03-21
+---
+
+## Abstract
+This is a way for users to safely store their nft's outside of their wallet.
+
+## Motivation
+Players over time will accumulate a large amount of assets through the collection of gaming assets as well as other general non-fungible tokens such as art. Wallets as they were originally conceived weren't meant to handle such a large volume of items and eventually they become cumbersome to organize.
+
+To solve this, an Nft Bank is necessary so that users may be able to securely store assets that may not currently be using, thus keeping their wallet manageable.
```

Comment:
make sure that "NFT" is capitalized throughout because it's an acronym for "non-fungible token"

Reply:
Also, add wallet security is not that user-friendly. Storing valuable NFTs in a hot wallet without backup account access as well as locking features is not conducive to secure asset storage.